ext/AbstractFFTsTestExt.jl

```diff
@@ -61,7 +62,9 @@ function TestUtils.test_plan(P::AbstractFFTs.Plan, x::AbstractArray, x_transform
         _x_out = similar(P * _copy(x))
         @test mul!(_x_out, P, _copy(x)) ≈ x_transformed
         @test _x_out ≈ x_transformed
-        @test P * view(_copy(x), axes(x)...) ≈ x_transformed # test view input
+        if test_wrappers
+            @test P * view(_copy(x), axes(x)...) ≈ x_transformed # test view input
+        end
```

Comment:
It would be better to mark this as broken if it fails instead of disabling the test.

Reply:
To clarify, if `test_wrappers = true`, we should use `@test`, and if `test_wrappers = false`, we should use `@test_broken`?

Reply:
One issue with `@test_broken` is that if the test fails sometimes, but not always, we get an unexpected pass error, which seems a bit brittle for users of `test_wrappers=false`. For now, I changed the test to use the skipping mechanism instead of my own branching, but let me know if you have further thoughts